modules/ROOT/pages/create-endpoint.adoc

```diff
@@ -55,6 +55,32 @@ Partner Manager supports the following endpoint protocols:
 | AS2 | Receive, Send | N/A
 |===
 
+== Endpoint Runtime Services
+
+Using dynamic configurations, Partner Manager deploys generic endpoint services that can receive data from or send data to different endpoint configurations. Endpoint configurations of all supported protocols can be reused across various message flows and trading partners.
+
+Partner Manager’s deployment orchestration service inspects the message flow and associated configuration objects and determines which endpoint services to deploy or update based on whether or not any new endpoint configurations are added, updated, or removed.
+
+The number and type of endpoint services deployed to your runtime will depend on the transport protocols used to exchange data with your trading partners and backend systems.
+
+Each unique AS2 and HTTP and HTTPS receive endpoint configurations result in the deployment of their own runtime applications. You can view the Runtime application name corresponding to a given configuration in Partner Manager, from the endpoint detail page.
+
+SFTP Receive, SFTP Send, AS2 Send and HTTP(s) send endpoints are always managed within a single generic endpoint service for the protocol & direction.
+
+=== Endpoint Services
+[%header%autowidth.spread]
+|===
+|Protocol |Direction |Application Name Prefix
+|SFTP |Receive |b2b-inbound-sftp-<Runtime Application Name>
+|SFTP |Send |b2b-outbound-sftp-<Runtime Application Name>
+|AS2 |Receive |b2b-inbound-as2-<Runtime Application Name>
+|AS2 |Send |b2b-outbound-as2-<Runtime Application Name>
+|HTTP or HTTPS |Receive |b2b-inbound-http-<Runtime Application Name>
+|HTTP or HTTPS |Send |b2b-outbound-http-<Runtime Application Name>
+|===
+
+If your deployment target is CloudHub, You can assign static IP addresses to your endpoint applications connecting to external systems from xref:runtime-manager::managing-applications-on-cloudhub#static-ips[Runtime Manager], if your organization or your trading partners require allowlisting IP addresses. You can share the static IP address assigned to your endpoint application with your trading partners for them to allowlist the IP address, allowing the application to connect with their server.
```

Comment:
Update "assign static IP addressed" to link https://docs.mulesoft.com/runtime-manager/managing-applications-on-cloudhub#static-ips